Source/WebCore/platform/graphics/cg/ColorSpaceCG.cpp

```diff
@@ -157,62 +157,62 @@ std::optional<ColorSpace> colorSpaceForCGColorSpace(CGColorSpaceRef colorSpace)
     // the reset in alphabetical order.
     // FIXME: Consider using a UncheckedKeyHashMap (with CFHash based keys) rather than the linear set of tests.
 
-    if (CGColorSpaceEqualToColorSpace(colorSpace, sRGBColorSpaceRef()))
+    if (CGColorSpaceEqualToColorSpace(colorSpace, RetainPtr { sRGBColorSpaceRef() }.get()))
```

Comment:
Better to rename `sRGBColorSpaceRef` to `sRGBColorSpaceSingleton`. Then static analysis will acknowledge that you're returning a global, and you won't need the `RetainPtr`.

Reply:
Same comment in many other places.